### ISSUE TITLE
Issue of generating only low-resolution DEMs

### ISSUE THREAD

**Issue by silverbean-j** [NeoGeographyToolkit/StereoPipeline#411]
Hello ASP Team,

I am using ASP to generate DEMs. However, I've encountered an issue where only low-resolution DEMs are being created, and I'm unable to generate high-resolution DEMs.

The command I'm using is as follows: parallel_stereo > point2dem. However, in the resulting files, I only get PC_sub files with low resolution, and the high-resolution PC files are missing.

I initially conducted tests using ASTER satellite imagery, but I encountered the same issue with other satellite imagery that I would like to work with personally.

Here are my system details:

Operating System: Ubuntu Linux

```
ASP Version:
NASA Ames Stereo Pipeline 3.4.0-alpha
Build ID: 1a50ea02
Build date: 2023-09-16

Built against:
NASA Vision Workbench 3.4.0-alpha
Build ID: 5088e058
USGS ISIS 8.0.0
Boost C++ Libraries 1.72.0
GDAL 3.5.3 | 20221027
```
I hope you can guide me if I'm using the wrong approach causing this bug or if there's a solution to enable the creation of high-resolution DEMs. Your assistance is greatly appreciated.

Thank you for taking the time to read this.

Sincerely,

**Comment by silverbean-j** [comment]
Hello,

Fortunately, I was able to successfully create a DEM using Aster satellite imagery by revisiting your manual. However, I'm still encountering the same issue when trying to create a DEM for the satellite imagery I want to produce personally.

This is the log file for the specific satellite:

While the multi-spectral image with a GSD of 2m is being generated without any resolution issues, the panchromatic image with a GSD of 0.5m is only producing low-resolution results.

For your reference, the specifications of the satellite I'm using are similar to PlanetScope, and the size of the panchromatic image is approximately 35,000x25,000.

The command I used is as follows:

`parallel_stereo -t rpc --stereo-algorithm asp_mgm --subpixel-mode 9 --alignment-method affineepipolar img01_P.tif img02_P.tif img01_Aux.xml img02_Aux.xml C4try03/try03 | tee stereo_output.log`

Thank you very much for your response.

**Comment by oleg-alexandrov** [comment]
A satellite like PlanetScope is not great for stereo, because, as I recall,
the images mostly face forward, so there is no baseline. Stereo needs
different perspectives. (
https://stereopipeline.readthedocs.io/en/latest/examples/stereo_pairs.html)

I see no log of how you ran the tool. By that, what is meant is a text file
having all the text that stereo produced in the terminal.

I suggest looking at all that text and searching for convergence angle. If
it tells you it is very small, like under 3 degrees, that is not good.
Here's a sample of such text:

Convergence angle percentiles (in degrees) based on interest point matches:
25% 42.4481, 50% 43.1319, 75% 44.0485.

You can also search for it in the output directory, in the log file whose
name is like run/run-log-stereo_pprc*.txt

In short, in your case the problem is likely with the data.

You will likely have more luck with Planet SkySat data. That we support
well, but there are many images that need to be processed together, which
makes the processing harder. (
https://stereopipeline.readthedocs.io/en/latest/examples/skysat.html)
